### ISSUE TITLE
[Feature request] Bring back "Activate first tab opened from current tab"

### ISSUE THREAD

**Issue by frontierpsycho** [ChaosinaCan/ClassicTabs#13]
Opera 12 has an option, when closing a tab, to activate the first tab opened from the current tab. That's very useful. Any plan to bring that back?

**Comment by frontierpsycho** [comment]
I think the difference between the default Opera 26 behaviour and the old Opera 12 behaviour is that in 26, if I open a bunch of background tabs from tab A, and then I close tab A, it doesn't focus the tabs I opened, but rather the tab in the left of tab A. 

However, it does indeed go through the tabs I opened from tab A, if I focus the first one, and then successively close them (I hope that was clear :) ).

**Comment by OpUser38** [comment]
> I think the difference between the default Opera 26 behaviour and the old Opera 12 behaviour
> is that in 26, if I open a bunch of background tabs from tab A, and then I close tab A,
> it doesn't focus the tabs I opened, but rather the tab in the left of tab A.
Absolutely. And here is a problem. Opera 12 behaviour should be copied in your extension for all happiness.
